build-system/amp4test.js

```diff
@@ -19,6 +19,17 @@ const app = module.exports = require('express').Router();
 
 app.use('/compose-doc', function(req, res) {
   res.setHeader('X-XSS-Protection', '0');
+  const mode = process.env.SERVE_MODE == 'compiled' ? '' : 'max.';
+  const extensions = req.query.extensions;
+  let extensionScripts = '';
+  if (!!extensions) {
+    extensionScripts = extensions.split(',').map(function(extension) {
+      return '<script async custom-element="'
+              + extension + '" src=/dist/v0/'
+              + extension + '-0.1.' + mode + 'js></script>';
```

Comment:
Sorry. Fail to notice this the other day. I see you hard code `0.1` to the script path. This would not work with all extensions. Please also specify version number. 
I am OK if you address this in separate PR. Let me know